tests/nested/manual/core-broken-fde-state/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: Verify that inconsistent FDE state generates warning on reseal
+
+details: |
+    When the primary key digests or the revocation counter handles
+    from the FDE state do not match what we get from resealing, we
+    should at least warn about it in the logs.
+
+systems:
+  - -ubuntu-1*
+  - -ubuntu-20*
+  - -ubuntu-22*
+
+prepare: |
+    tests.nested build-image core
+    tests.nested create-vm core
+
+    unsquashfs -d pc-kernel "$(tests.nested get assets-path)"/pc-kernel*.snap
+    quiet apt install -y systemd-boot-efi systemd-ukify
+
+    objcopy -O binary -j .initrd pc-kernel/kernel.efi initrd.img
+    objcopy -O binary -j .linux pc-kernel/kernel.efi linux
+
+    /usr/lib/systemd/ukify build --linux=linux --initrd=initrd.img --section=.spread:test --output=pc-kernel/kernel.efi
+
+    KEY_NAME=$(tests.nested download snakeoil-key)
+    SNAKEOIL_KEY="$PWD/$KEY_NAME.key"
+    SNAKEOIL_CERT="$PWD/$KEY_NAME.pem"
+    tests.nested secboot-sign file "$PWD/pc-kernel/kernel.efi" "$SNAKEOIL_KEY" "$SNAKEOIL_CERT"
+
+    snap pack pc-kernel --filename=new-pc-kernel.snap
+    remote.push new-pc-kernel.snap
```

Comment:
Why do we actually need to create a new kernel snap? Would not the warnings happen also if we reinstall the already installed kernel, as we are corrupting the FDE state?

Reply:
Got it, thanks. Maybe good to have a comment here.

Reply:
Done.